### ISSUE TITLE
Updated version of contractsURI

### ISSUE THREAD

**Issue by thelastjosh** [metagov/daostar#231]
Re: discussion with Arnold of Lighthouse (DAO aggregator) on May 29, 2024.

This is a stub for a new standard that develops out what we would like to see in contractsURI (currently an optional subfield of daoURI) and how we should go about verifying self-reported information in contractsURI.

We want ideally on-chain verification of the following list of data:
1. On-chain contracts
2. Associated treasuries, e.g. Safes
3. Official Telegram
4. Official Discord

All this information stems from deployer contract? Could we talk to Hats about integrating some of their on-chain information into daoURI?

## Motivation
1. Aggregator
2. Trustless / permissionless integrations (i.e. more interop)
3. Audits

Also see discussion of DAO ID: #213 .

**Comment by thelastjosh** [comment]
No update, though Josh could try to follow up with Arnold about putting this together?

**Comment by thelastjosh** [comment]
- [ ] @amanwithwings will touch base with Arnold and share this link.

**Comment by Rashmi-278** [comment]
Would be great to combine this with https://uma.xyz/#how-it-works

**Comment by 1a35e1** [comment]
Hi @thelastjosh , @amanwithwings. Sorry I missed today. I had a call overlap that took precedence. 

Happy to schedule a meeting ahead of next community call so we can scope this out further. Will coordinate with @amanwithwings

**Comment by 1a35e1** [comment]
> Would be great to combine this with https://uma.xyz/#how-it-works

Nice find. This looks super interesting. I will have to dive deeper into the mechanics of how this works.

**Comment by Rashmi-278** [comment]
> > Would be great to combine this with https://uma.xyz/#how-it-works
> 
> Nice find. This looks super interesting. I will have to dive deeper into the mechanics of how this works.

Sounds Good, I'm thinking UMA Protocol would play a role in Verification of the contractURI's published, If we want Trustless / permissionless integrations (i.e. more interop) UMA could be used as a permissionless verifier where incorrect  URI's would be disputed

**Comment by 1a35e1** [comment]
Some interesting Mission Requests that recently got published from OP that fold into this request

* https://github.com/ethereum-optimism/ecosystem-contributions/issues/207
* https://github.com/ethereum-optimism/ecosystem-contributions/issues/216

**Comment by thelastjosh** [comment]
Some minor updates last week between Arnold and Rashmi.

Waiting on rough implementation spec from Arnold.

Question: how does an auditor like Chainanalysis verify whether a given contract "belongs to" a given DAO or is under its control? Is a protocol-verification strategy too far / too hard for this problem; should we be pursuing something simpler, based on the assumption that we will still need active auditors?

Would be nice to get on a call with an auditor type entity.
